README.md

```diff
@@ -45,21 +49,19 @@ Example: `-o cloudfour-a.csv -o google-sheets -o google-sheets:"Spreadsheet Name
 
 #### `--ignore-robots-txt`
 
-Crawl pages even if they are listed in the site's robots.txt (default false)
+Crawl pages even if they are listed in the site's `robots.txt`
 
 #### `--crawler-user-agent`
 
 Pass a user agent string to be used by the crawler (not by Lighthouse)
 
 #### `--lighthouse-concurrency`
 
-Control the maximum number of Lighthouse reports running concurrently (default 7)
+Control the maximum number of Lighthouse reports running concurrently (defaults to `os.cpus().length - 1`)
 
-#### `--lh:only-categories`
+#### `--lighthouse-category`
```

Comment:
I switched this flag to be pass-multiple-times-for-multiple-values instead of passing it once with a comma-separated values. This makes it consistent with the other flags, but it makes it inconsistent with Lighthouse CLI.

I am still not sure if this is the right change and I'd like to hear what other people think.

Reply:
Hmm…I think at this point I would prefer consistency with other flags. 

If this becomes an issue, we can modify it later. I say keep this change. 👍🏽